cf-macro/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const _ = require('lodash')
+const Ajv = require('ajv')
+
+const alarms = require('slic-watch-core/alarms')
+const dashboard = require('slic-watch-core/dashboard')
+const CloudFormationTemplate = require('slic-watch-core/cf-template')
+const defaultConfig = require('slic-watch-core/default-config')
+const { slicWatchSchema } = require('slic-watch-core/config-schema')
+
+exports.handler = async function (event, context) {
+  const response = processFragment(event)
+  return response
+}
+
+function processFragment (event) {
+  let status = 'success'
+  let outputFragment = event.fragment
+  try {
+    const slicWatchConfig = (outputFragment.Metadata || {}).slicWatch || {}
+    const ajv = new Ajv({
+      unicodeRegExp: false
+    })
+    const config = _.merge(defaultConfig, slicWatchConfig)
```

Comment:
Are we doing this because we need to support recursive merge?

Otherwise, we could have just done:

```js
const config = {...defaultConfig, ...slicWatchConfig}
```

Reply:
I believe we need recursive merge.  This was copied from Serverless Plugin implementation.